common/src/main/java/org/apache/drill/common/scanner/ClassPathScanner.java

```diff
@@ -422,11 +423,16 @@ static ScanResult scan(Collection<URL> pathsToScan, Collection<String> packagePr
     }
   }
 
-  private static void verifyClassUnicity(List<AnnotatedClassDescriptor> annotatedClasses) {
+  private static void verifyClassUnicity(List<AnnotatedClassDescriptor> annotatedClasses, Collection<URL> pathsScanned) {
     Set<String> scanned = new HashSet<>();
     for (AnnotatedClassDescriptor annotated : annotatedClasses) {
       if (!scanned.add(annotated.getClassName())) {
-        throw new RuntimeException("BUG: " + annotated.getClassName() + " scanned twice");
+        throw UserException.functionError()
+            .message(
+                "function %s scanned twice in %s\nDo you have conflicting jars on the classpath?",
+                annotated.getClassName(), pathsScanned
```

Comment:
I put the URLs on a separate line.
it's array.toString() which is unhelpful.
Arrays.toString(array) looks the same as collection.toString()